tests/unit/test_exceptions.py

```diff
@@ -14,35 +14,37 @@
 
 import http.client
 import json
+from google.rpc import status_pb2
+from google.rpc.status_pb2 import Status
 
 import grpc
 import mock
 import requests
-
+from google.rpc import error_details_pb2
+from google.rpc import code_pb2
+from google.protobuf import any_pb2, json_format
+from grpc_status import rpc_status
 from google.api_core import exceptions
-
+from google.protobuf import text_format
 
 def test_create_google_cloud_error():
     exception = exceptions.GoogleAPICallError("Testing")
     exception.code = 600
-    assert str(exception) == "600 Testing"
+    assert str(exception) == "600 Testing []"
```

Comment:
As above, the empty list is unhelpful.  We also need a test which exercises the `__str__` method when there *are* error details present.

Reply:
Done. I added a check in a different test later in the code.